.openshift-ci/jobs/update-support-packages/env.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -x
```

Comment:
Is it still needed, or a leftover from debugging?

Reply:
The env.sh file for integration tests uses "set -x". I kept it here for consistency. I could delete it in both places. It seems useful to have -x here so that if something does need to be debugged it can be done easily.

Reply:
It (`set -x`) can lead to credential leakage. I'd remove it from both.

Reply:
> It (set -x) can lead to credential leakage. I'd remove it from both.

Yep, I have the same concerns.